### PR TITLE
Fix sitemap opportunity

### PR DESCRIPTION
**Pull request by dzehnder**
The suggestions are split into two parts:

Errors concerning fetching sitemap or robots file: 
```
{
   opportunityId: 'id',
   type: 'REDIRECT_UPDATE',
   rank: 0,
   data: {
      type: 'error',
      error: 'NO VALID URLs FOUND IN SITEMAP',
   }
}
```

Errors concerning urls in the sitemap:
```
{
   opportunityId: 'id',
   type: 'REDIRECT_UPDATE',
   rank: 0,
   data: {
      type: 'url',
      sitemapUrl: 'https://some-domain.adobe/sitemap.xml',
      pageUrl: 'https://some-domain.adobe/foo',
      statusCode: 404,
   }
}
```

### CODE REVIEW

File: src/sitemap/handler.js [hunk 1]
```diff
@@ -395,182 +399,135 @@ export async function sitemapAuditRunner(baseURL, context) {
...
 
 export async function convertToOpportunity(auditUrl, auditData, context) {
   const { dataAccess, log } = context;
 
-  const classifiedOpportunities = await classifyOpportunities(auditUrl, auditData);
-  if (!classifiedOpportunities.length) {
+  log.info('Converting SITEMAP audit to opportunity...');
+
+  const classifiedSuggestions = classifySuggestions(auditUrl, auditData, log);
+  if (!classifiedSuggestions.length) {
```
Comment (sandsinh) [inline src/sitemap/handler.js:479]: Should we also delete an existing opportunity or remove suggestions if there are no classifiedSuggestions in an audit ?

Comment (dzehnder) [reply src/sitemap/handler.js:479]: I don't think we are doing that right now for any opportunity. I think that decision should be made on PO level

Comment (sandsinh) [reply src/sitemap/handler.js:479]: okay.

**Review by sandsinh — APPROVED** [review]

**Pull request merged**